### ISSUE TITLE
Allow filtering by Company, Country or others

### ISSUE THREAD

**Issue by jslawler** [AdguardTeam/AdGuardDNS#517]
Example 1: User views the query lock by {Company - Apple} + {Blocked} and decides they want to unblock them all.

Example 2: User decides they don't trust Google at all, and they want to block everything from {Company - Google}.

Example 3: User decides they don't want any Chinese domains to be queried from any of their devices.

In examples such as these, it would be nice to have an option in the to add all all of a companies and countries domains to the the user list, or something similar.

As we can already filter by these types, it makes sense that we should be able to filter by these types as well.

We can already do this by using domain filter rules, but it would be good to have a human friendly interface to do it with.
